README.md

```diff
@@ -52,10 +52,14 @@ Formerly known as Real-Time For the Masses.
 
 - Applications must be written using the 2018 edition.
 
-## [User documentation](https://rtic.rs)
+## [User documentation](https://rtic.rs) [(Development version)](https://rtic.rs/dev)
```

Comment:
have not looked at the rendered version yet but adding something that's not a link like a dash (-) would help visually identify this line as 2 separate links; otherwise it may look like a single link.

Reply:
That is a good point!